bazel/linux/defs.bzl

```diff
@@ -991,3 +991,31 @@ def kernel_test(name, kernel_image, module, rootfs_image = None, test_dir_cfg =
     if rootfs_image:
         cfg = mconfig(cfg, rootfs_image = rootfs_image)
     name_runner = mcreate_rule(name, runner, "", runner_cfg, kwargs, cfg)
+
+def _bundle(ctx):
```

Comment:
minor nit: the current naming convention in the file is unfortunate, a bundle is something else (a kernel module compiled against different archs/kernel - a bundle of the same kernel module).

Right now, bundles are called "runtimes", I'd like to move them (this, and the only other one defined already) into a runtimes.bzl, and probably do a cleanup sweep to rename all rules to decouple from kernel / have more sensible names.

(really: some of the stuff we're adding now has to do with running in an emulator, not much with kernel builds)